cedar-lean/Cedar/Data/LT.lean

```diff
@@ -145,36 +145,36 @@ theorem List.lt_conn [LT α] [StrictLT α] {xs ys : List α} :
   intro h₁
   by_contra h₂
   simp [not_or] at h₂
-  rcases h₂ with ⟨h₂, h₃⟩
+  have ⟨h₂, h₃⟩ := h₂
   cases xs <;> cases ys
-  case intro.nil.nil => contradiction
-  case intro.nil.cons _ _ xhd xtl =>
-    rcases (List.lt.nil xhd xtl) with h₄
+  case nil.nil => contradiction
+  case nil.cons _ _ xhd xtl _ =>
+    have h₄ := List.lt.nil xhd xtl
     contradiction
-  case intro.cons.nil _ _ yhd ytl =>
-    rcases (List.lt.nil yhd ytl) with h₄
+  case cons.nil _ _ yhd ytl _ =>
```

Comment:
Adding the extra underscore to pad out to the total # of arguments?

Reply:
This is because of the shadowing behavior change discussed in the PR description.  I think `have` results in the shadowed hypothesis being in a different position in the list, which affects what gets bound here.  Adding this underscore ensures that `yhd` and `ytl` are bound to the same things that they were previously, before the change.